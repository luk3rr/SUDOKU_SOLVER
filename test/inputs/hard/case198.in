200090001 005060002 370001409 000000008 000000520 050002094 000900700 031000000 007654000 
