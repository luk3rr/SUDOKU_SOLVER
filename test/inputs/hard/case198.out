Solving the following grid:
2 0 0 | 0 9 0 | 0 0 1 
0 0 5 | 0 6 0 | 0 0 2 
3 7 0 | 0 0 1 | 4 0 9 
------+-------+------
0 0 0 | 0 0 0 | 0 0 8 
0 0 0 | 0 0 0 | 5 2 0 
0 5 0 | 0 0 2 | 0 9 4 
------+-------+------
0 0 0 | 9 0 0 | 7 0 0 
0 3 1 | 0 0 0 | 0 0 0 
0 0 7 | 6 5 4 | 0 0 0 


Solution found :')

2 8 4 | 3 9 5 | 6 7 1 
9 1 5 | 4 6 7 | 8 3 2 
3 7 6 | 2 8 1 | 4 5 9 
------+-------+------
7 2 3 | 5 4 9 | 1 6 8 
1 4 9 | 8 3 6 | 5 2 7 
6 5 8 | 1 7 2 | 3 9 4 
------+-------+------
4 6 2 | 9 1 3 | 7 8 5 
5 3 1 | 7 2 8 | 9 4 6 
8 9 7 | 6 5 4 | 2 1 3 

Algorithm: BFS
Total time: 834 ms
Total expanded states: 120763
